docs/maps/clean-data.asciidoc

```diff
@@ -0,0 +1,183 @@
+[role="xpack"]
+[[maps-clean-your-data]]
+=== Cleaning your data
```

Comment:
Can you change 'Cleaning' to 'Clean' to match title of 'Import geo data'

Reply:
Sorry, missed this one :facepalm:, fixed at 6f54ed4